WordPress/WordPressNotificationContentExtension/Sources/Tracks/Tracks+ContentExtension.swift

```diff
@@ -6,8 +6,8 @@ import Foundation
 /// - launched: the content extension was successfully entered & launched
 ///
 private enum ContentExtensionEvents: String {
-    case launched           = "wpios_notification_content_extension_launched"
-    case failedToMarkAsRead = "wpios_notification_content_extension_failed_mark_as_read"
+    case launched           = "notification_content_extension_launched"
+    case failedToMarkAsRead = "notification_content_extension_failed_mark_as_read"
```

Comment:
I tested the content extension by receiving a notification with an image in it and long-pressing to view the image. (FWIW these were added in https://github.com/wordpress-mobile/WordPress-iOS/pull/18048). I couldn't seem to attach Xcode to `WordPressNotificationContentExtension` (note there's no `JetpackNotificationContentExtension` – @hassaanelgarem).

@hassaanelgarem, do you know if there should be a separate `JetpackNotificationContentExtension`? It seems to work fine without it, but not sure if it's intentional.

I don't think this is a blocker to this PR since the extension works fine from a user's perspective and most sites don't support images in comments. P2s are an exception to this (see https://wordpress.com/p2/) so this won't be widely used.

Reply:
> Quoted from https://github.com/wordpress-mobile/WordPress-iOS/pull/18048's description.

Good find! I should have read the PR description 😄 

> My understanding is that `WordPressNotificationContentExtension` can be safely removed.

I created an issue to track the removal of this: https://github.com/wordpress-mobile/WordPress-iOS/issues/19535